EventKit/eventkit-cloud
# adds script to check code

## Description of Improvement

Adds a script to run linters and tests concurrently.  It doesn't do formatting in order to prevent modifications to code while tests are loading. 

## How to test

Run `python scripts/check_code.py`.  If there are no errors, nothing should print.  Try adding an error to the code, and running again, to ensure that the results are printed to the terminal when needed. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
